cmd/slinky-config/main.go

```diff
@@ -375,7 +375,13 @@ func createOracleConfig() error {
 	// add raydium provider to the list of providers if enabled
 	if raydiumEnabled {
 		cfg := raydium.DefaultAPIConfig
-		cfg.URL = solanaNodeURL
+		nodes := strings.Split(solanaNodeURLs, ",")
+
+		for _, node := range nodes {
+			cfg.Endpoints = append(cfg.Endpoints, config.Endpoint{
+				URL: node,
+			})
+		}
```

Comment:
The solanaNodeURLs flag should just be a [StringSliceVarP](https://pkg.go.dev/github.com/spf13/pflag#FlagSet.StringSliceVarP). It eliminates all of this code.